### PR TITLE
Admin panel users and orgs

### PR DESCRIPTION
**Pull request by swampie**
This PR adds the documentation for the Admin Panel and User/Organization management

### CODE REVIEW

**Review by llewellyn-sl — CHANGES_REQUESTED** [review]
I edited the overview.md directly with some grammatical changes. Please rename `ysers_organizations.png` to `users_organizations.png`. Other than that, LGTM.

File: docs/administration/overview.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Administration of Users, Organizations, and Memberships
+
+As a **Root User**, you can access a comprehensive overview of the users and organizations in the system from the **Admin Panel**.
+
+The Admin Panel menu entry will only be accessible in the top right avatar menu if you are logged in as a Root user. This role should only be assigned to a system administrator, since it enables several high level and potentially risky operations.
+
+## Users Administration
```
Comment (llewellyn-sl) [inline docs/administration/overview.md:7]:
```suggestion
## Users administration
```

File: docs/administration/overview.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+![](_images/manage_orgs_button.png)
+
+The Users administration page lists all the users in the Tower database. From this page, you can:
+
+### Search Users
+The user search function allows you to find a specific user by name or email and perform various operations with that user.
+
+### Create a User
+The Add User button in the top right corner above the table allows you to create a new user from scratch. If the new user email already exists in the system, the user creation will fail. Once the new user has been created, inform them that access has been granted.
```
Comment (llewellyn-sl) [inline docs/administration/overview.md:17]:
```suggestion
The Add user button above the table allows you to create a new user from scratch. If the new user email already exists in the system, the user creation will fail. Once the new user has been created, inform them that access has been granted.
```

File: docs/administration/overview.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+### Edit a User
+
+![](_images/edit_user.png)
+
+By selecting a username from the table, you can edit some of the user details, or delete the user.
```
Comment (llewellyn-sl) [inline docs/administration/overview.md:23]:
```suggestion
By selecting a username from the table, you can edit the user's details, or delete the user.
```

File: docs/administration/overview.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+
+### Membership Administration 
+
+**Available from version 22.3.X**
+
+![](_images/organization_members.png)
+
+From the user list, you have an overview of all the memberships for a selected user. The Membership administration page can be reached by selecting the **Edit Organizations** button. From here, you can list and search for all the organizations the user belongs to (as a member or as an owner), change the role of the user for a given membership, remove the user from an organization, and add the user to a new organization.
```
Comment (llewellyn-sl) [inline docs/administration/overview.md:32]:
```suggestion
From the user list, you have an overview of all the memberships for the selected user. The Membership administration page is reached by selecting the **Edit organizations** button. From here, you can list and search for all the organizations the user belongs to (as a member or as an owner), change the role of the user for a given membership, remove the user from an organization, or add the user to a new organization. 
```

File: docs/administration/overview.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+**Available from version 22.3.X**
+
+![](_images/organization_members.png)
+
+From the user list, you have an overview of all the memberships for a selected user. The Membership administration page can be reached by selecting the **Edit Organizations** button. From here, you can list and search for all the organizations the user belongs to (as a member or as an owner), change the role of the user for a given membership, remove the user from an organization, and add the user to a new organization. 
+
+**Note:** You can only add users to an existing organization, and you cannot remove the last owner of an organization.
+
+## Organizations Administration
```
Comment (llewellyn-sl) [inline docs/administration/overview.md:36]:
```suggestion
## Organization administration
```

File: docs/administration/overview.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+![](_images/organization_administration.png)
+
+The Organization administration page lists all the organizations in the Tower database. From this page, you can:
+
+### Search Organizations
```
Comment (llewellyn-sl) [inline docs/administration/overview.md:42]:
```suggestion
### Search organizations
```

File: docs/administration/overview.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+The organization search function allows you to find a specific organization by its name or email and perform various operations with that organization.
+
+### Create an Organization
```
Comment (llewellyn-sl) [inline docs/administration/overview.md:45]:
```suggestion
### Create an organization
```

File: docs/administration/overview.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+The Create Organization button in the top right corner above the table allows you to create a new organization from scratch.
```
Comment (llewellyn-sl) [inline docs/administration/overview.md:46]:
```suggestion
The Create organization button above the table allows you to create a new organization from scratch.
```

File: docs/administration/overview.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+
+### Edit an Organization
```
Comment (llewellyn-sl) [inline docs/administration/overview.md:48]:
```suggestion
### Edit an organization
```

File: docs/administration/overview.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+
+![](_images/edit_organization.png)
+
+By selecting an organization name from the table, you can edit some of the organization details, or delete it.
```
Comment (llewellyn-sl) [inline docs/administration/overview.md:52]:
```suggestion
By selecting an organization name from the table, you can edit the organization's details, or delete it.
```

File: docs/administration/overview.md [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+
+
+### Membership Administration
```
Comment (llewellyn-sl) [inline docs/administration/overview.md:55]:
```suggestion
### Membership administration 
```

File: docs/administration/overview.md [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+By selecting an organization name from the table, you can edit some of the organization details, or delete it.
+
+
+### Membership Administration 
+
+**Available from version 22.3.X**
+
+![](_images/organization_members.png)
+
+From the organizations list, you have an overview of all the memberships for the selected organization. Select the **Manage User** button to access the Membership administration page. From here, you can list and search for all the users that are members or owners of the selected organization, change the role of the user for the given membership, remove the member from the organization, and add a new user to the organization.
```
Comment (llewellyn-sl) [inline docs/administration/overview.md:61]:
```suggestion
From the organizations list, you have an overview of all the memberships for the selected organization. Select the **Manage users** button to access the Membership administration page. From here, you can list and search for all the users that are members or owners of the selected organization, change the role of the user for the given membership, remove the member from the organization, or add a new user to the organization.
```

Comment (llewellyn-sl) [inline docs/administration/overview.md:61]:
```suggestion
From the organizations list, you have an overview of all the memberships for the selected organization. Select the **Manage users** button to access the Membership administration page. From here, you can list and search for all the users that are members or owners of the selected organization, change the role of the user for the given membership, remove the member from the organization, or add a new user to the organization.
```

File: mkdocs.yml [hunk 13]
```diff
@@ -93,6 +93,7 @@ nav:
       - 'Organizations': 'orgs-and-teams/organizations.md'
       - 'Workspace Management': 'orgs-and-teams/workspace-management.md'
       - 'Shared Workspaces': 'orgs-and-teams/shared-workspaces.md'
+      - 'Users and Organizations Administration': 'administration/overview.md'
```
Comment (llewellyn-sl) [inline mkdocs.yml:96]:
```suggestion
      - 'User and organization administration': 'administration/overview.md'
```

File: mkdocs.yml [hunk 14]
```diff
@@ -93,6 +93,7 @@ nav:
       - 'Organizations': 'orgs-and-teams/organizations.md'
       - 'Workspace Management': 'orgs-and-teams/workspace-management.md'
       - 'Shared Workspaces': 'orgs-and-teams/shared-workspaces.md'
```
Comment (llewellyn-sl) [inline mkdocs.yml:95]:
```suggestion
      - 'Shared workspaces': 'orgs-and-teams/shared-workspaces.md'
```

File: mkdocs.yml [hunk 15]
```diff
@@ -93,6 +93,7 @@ nav:
       - 'Organizations': 'orgs-and-teams/organizations.md'
       - 'Workspace Management': 'orgs-and-teams/workspace-management.md'
```
Comment (llewellyn-sl) [inline mkdocs.yml:94]:
```suggestion
      - 'Workspace management': 'orgs-and-teams/workspace-management.md'
```

File: mkdocs.yml [hunk 16]
```diff
@@ -93,6 +93,7 @@ nav:
...
       - 'Shared Workspaces': 'orgs-and-teams/shared-workspaces.md'
+      - 'Users and Organizations Administration': 'administration/overview.md'
     - 'Launching Pipelines':
       - 'Launchpad': 'launch/launchpad.md'
       - 'Launch Form': 'launch/launch.md'
```
Comment (llewellyn-sl) [inline mkdocs.yml:99]:
```suggestion
      - 'Launch form': 'launch/launch.md'
```

File: docs/administration/overview.md [hunk 17]
```diff
@@ -0,0 +1,63 @@
...
+
+The Admin Panel menu entry will only be accessible in the top right avatar menu if you are logged in as a Root user. This role should only be assigned to a system administrator, since it enables several high level and potentially risky operations.
+
+## Users Administration
+
+![](_images/manage_orgs_button.png)
+
+The Users administration page lists all the users in the Tower database. From this page, you can:
+
+### Search Users
```
Comment (llewellyn-sl) [inline docs/administration/overview.md:13]:
```suggestion
### Search users
```

File: docs/administration/overview.md [hunk 18]
```diff
@@ -0,0 +1,63 @@
...
+The user search function allows you to find a specific user by name or email and perform various operations with that user.
+
+### Create a User
+The Add User button in the top right corner above the table allows you to create a new user from scratch. If the new user email already exists in the system, the user creation will fail. Once the new user has been created, inform them that access has been granted.
+
+### Edit a User
```
Comment (llewellyn-sl) [inline docs/administration/overview.md:19]:
```suggestion
### Edit a user
```

File: docs/administration/overview.md (hunk 4 above)
Comment (llewellyn-sl) [inline docs/administration/overview.md:32]:
```suggestion
From the user list, you have an overview of all the memberships for a selected user. The Membership administration page can be reached by selecting the **Edit organizations** button. From here, you can list and search for all the organizations the user belongs to (as a member or as an owner), change the role of the user for a given membership, remove the user from an organization, and add the user to a new organization. 
```

File: mkdocs.yml [hunk 19]
```diff
@@ -91,11 +91,12 @@ nav:
...
-      - 'Workspace Management': 'orgs-and-teams/workspace-management.md'
-      - 'Shared Workspaces': 'orgs-and-teams/shared-workspaces.md'
+      - 'Workspace management': 'orgs-and-teams/workspace-management.md'
+      - 'Shared workspaces': 'orgs-and-teams/shared-workspaces.md'
+      - 'User and organization administration': 'administration/overview.md'
     - 'Launching Pipelines':
       - 'Launchpad': 'launch/launchpad.md'
-      - 'Launch Form': 'launch/launch.md'
+      - 'Launch form': 'launch/launch.md'
       - 'Advanced Options': 'launch/advanced.md'
```
Comment (llewellyn-sl) [inline mkdocs.yml:100]:
```suggestion
      - 'Advanced options': 'launch/advanced.md'
```

**Review by llewellyn-sl — APPROVED** [review]
Minor changes. Looks good.

**Comment by llewellyn-sl** [comment]
@swampie can we fix this screen shot, please? I'm not a root user so I can't view and edit user pages, but we need a shot with a user that doesn't include Paolo's personal Gmail address :)
image

**Comment by llewellyn-sl** [comment]
Same goes for edit_user.png — we can either do new screens for this, or redact the Gmail address in the existing screens. @swampie your call.

**Comment by pditommaso** [comment]
Also, screenshot should use neutral avatar. Not employees ones.

**Comment by llewellyn-sl** [comment]
> Also, screenshot should use neutral avatar. Not employees ones.

I'm adding a task to my auditing to-do list to clean up screen shots across help.tower.nf for consistency and privacy.

**Pull request merged**